.gitignore

```diff
@@ -5,7 +5,9 @@
 osc-server-bom/root/opt/vmidc/bin/webapp/**
 osc-export/generated/**
 osc-export/webapp/**
-osc-export/data/**
+osc-export/data/*.db*

```

Comment:
how about the keystore/truststore etc files

Reply:
We check in keystore/truststore files.

Reply:
yes. But when those files change because of certificates added to them etc...they should be ignored.